MapleServer2/Types/IInventory.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Maple2Storage.Enums;
+using MapleServer2.Servers.Game;
+
+namespace MapleServer2.Types;
+
+public interface IInventory
+{
+    Dictionary<InventoryTab, short> ExtraSize { get; }
+    long Id { get; }
+    Dictionary<ItemSlot, Item> Equips { get; }
+    Dictionary<ItemSlot, Item> Cosmetics { get; }
+    Item[] Badges { get; }
+    Item[] LapenshardStorage { get; }
+    Dictionary<long, Item> TemporaryStorage { get; }
```

Comment:
I think if you remove all of these except Id, we could use this interface for the others inventories. Maybe in the future, not now.

Reply:
That's the plan! I wanted to keep this PR succinct, and then have other PRs to handle these.
IMHO they should be separated out, ie Inventory should not handle Equipment, but that's a discussion for later :P